GliaWidgets/View/Call/CallView.swift

```diff
@@ -7,8 +7,40 @@ class CallView: EngagementView {
         case upgrading
     }
 
+    var callDuration: String? {
+        didSet {
+            guard !isVisitrOnHold else { return }
+
+            secondLabel.text = callDuration
+            connectView.statusView.setSecondText(callDuration, animated: false)
+        }
+    }
+
+    var isVisitrOnHold: Bool = false {
+        didSet {
+            connectView.operatorView.isVisitorOnHold = isVisitrOnHold
+
+            if isVisitrOnHold {
+                secondLabel.text = style.onHoldTopText
+                connectView.statusView.setSecondText(style.onHoldTopText, animated: false)
+            }
+
+            if case .video = mode {
+                connectView.isHidden = !isVisitrOnHold
+                topStackView.isHidden = isVisitrOnHold
+                remoteVideoView.isHidden = isVisitrOnHold
```

Comment:
Is it possible to add test that will ensure that all theses flags are flipped as expected?

Reply:
Added tests